self_hosting_machinery/dashboards/utils.py

```diff
@@ -1,13 +1,9 @@
 import copy
-
 import pandas as pd
 
-from datetime import datetime
-from typing import Dict, Any, Optional
 from dataclasses import dataclass
-
-
-from self_hosting_machinery.webgui.selfhost_database import StatisticsService
+from datetime import datetime
+from typing import Dict, Any
 
 
 @dataclass
```

Comment:
not sure if I'd move it to selfhost_database  🤔

Reply:
))